fe/fe-core/src/main/java/com/starrocks/qe/VariableMgr.java

```diff
@@ -103,6 +103,8 @@ public class VariableMgr {
     public static final int READ_ONLY = 8;
     // Variables with this flag can not be seen with `SHOW VARIABLES` statement.
     public static final int INVISIBLE = 16;
+    // Variables with this flag will not forward to leader when modified in session
+    public static final int DISABLE_FORWARD_TO_LEADER = 1 << 5;
```

Comment:
Use affirmative tone instead
```suggestion
    public static final boolean ENABLE_FORWARD_TO_LEADER = false;
```

Reply:
This is used as a modifier for session variables. It can be set as a flag.